ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml

```
@@ -0,0 +1,80 @@
+periodics:
+- agent: kubernetes
+  cluster: vsphere
+  cron: 0 0 * * *
```

What about to run once a week at Sundays?
```suggestion
  cron: 0 0 * * 0
```